docs/docs/01-ibc/14-permissioning.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Permissioning
+sidebar_label: Permissioning
+sidebar_position: 14
+slug: /ibc/permissioning
+---
+
+# Permissioning
+
+IBC is designed at its base level to be a permissionless protocol. This does not mean that chains cannot add in permissioning on top of IBC. In ibc-go this can be accomplished by implementing and wiring an ante-decorator that checks if the ibc-message is signed by a permissioned authority. If the signer address check passes, the tx can go through otherwise it is rejected from the mempool.
+
+The antehandler runs before message-processing so it acts as a customizable filter that can reject messages before they get included in the block. The SDK allows developers to write antedecorators that can be stacked with others to add multiple independent customizable filters that run in sequence. Thus, chain developers that want to permission IBC messages are advised to implement their own custom permissioned IBC antedecorator to add to the standard antedecorator stack.
+
+## Best Practices
+
+**MsgCreateClient**: Permissioning the client creation is the most important for permissioned IBC. This will prevent malicious relayers from creating clients to fake chains. If a chain wants to control which chains are connected to it directly over IBC, the best way to do this is by controlling which clients gets created. The permissioned authority can create clients only of counterparties that the chain approves of. The permissioned authority can be the governance account, however MsgCreateClient contains a consensus state that can be expired by the time governance passes the proposal to execute the message. Thus, if the voting period is longer than the unbonding period of the counterparty; it is advised to use a permissioned authority that can immediately execute the transaction (e.g. a trusted multisig)
```

Comment:
```suggestion
**MsgCreateClient**: Permissioning the client creation is the most important for permissioned IBC. This will prevent malicious relayers from creating clients to fake chains. If a chain wants to control which chains are connected to it directly over IBC, the best way to do this is by controlling which clients gets created. The permissioned authority can create clients only of counterparties that the chain approves of. The permissioned authority can be the governance account, however `MsgCreateClient` contains a consensus state that can be expired by the time governance passes the proposal to execute the message. Thus, if the voting period is longer than the unbonding period of the counterparty; it is advised to use a permissioned authority that can immediately execute the transaction (e.g. a trusted multisig)
```